src/sempy_labs/report/_reportwrapper.py

```
@@ -1797,46 +1860,61 @@ def get_theme(self, theme_type: str = "baseTheme") -> dict:
 
-        theme_name = theme_file.get("name")
+        self._ensure_pbir()
+        theme_version = "5.6.4"
+
+        # Extract theme_json from theme_file_path
+        if theme_file_path:
+            # Open file
+            if not theme_file_path.endswith(".json"):
+                raise ValueError(
+                    f"{icons.red_dot} The '{theme_file_path}' theme file path must be a .json file."
+                )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)